app/src/androidTest/java/com/github/sdp/tarjetakuna/ui/ProfileFragmentInActivityTest.kt

```diff
@@ -25,6 +28,13 @@ class ProfileFragmentInActivityTest {
     private lateinit var activityRule: ActivityScenario<MainActivity>
     private val navController = TestNavHostController(ApplicationProvider.getApplicationContext())
 
+    @Rule
+    @JvmField
+    val grantPermissionRule: GrantPermissionRule = GrantPermissionRule.grant(
```

Comment:
Why do you need to put these lines in this fragment test?

Reply:
Because the location permission is in the MainActivity and so the permission request is called each time we use the MainActivity , a solution would be to create an entirely other fragment to implement the location permission but that seems overkill